Formula/minimesos.rb

```diff
@@ -0,0 +1,16 @@
+class Minimesos < Formula
+  desc "Testing infrastructure for Mesos frameworks"
+  homepage "https://minimesos.org/"
+  url "https://github.com/ContainerSolutions/minimesos/archive/0.10.2.tar.gz"
+  sha256 "16b198078c9384426f8849b08e92e173816c4d6626331bc6569aa5de89dffbdd"
+
+  depends_on "docker"
+
+  def install
+    bin.install "bin/minimesos"
+  end
+
+  test do
+    assert_match "Minimesos requires Docker 1.11.0 or higher", shell_output("#{bin}/minimesos")
```

Comment:
Test is failing: https://bot.brew.sh/job/Homebrew%20Core/10466/version=any_sierra/testReport/junit/brew-test-bot/sierra/test_minimesos/

Perhaps just `assert_match "Docker", shell_output("#{bin}/minimesos 2>&1")` is the best bet?